### PR TITLE
Core Branding Add New Icons

### PR DESCRIPTION
**Pull request by grgia**
### Details

Replace Icons in New Dot for WAQ Branding

### Fixed Issues

$ https://github.com/Expensify/App/issues/12442

### Tests

- [x] Verify that no errors appear in the JS console
- [x] Verify all icons look correct

### QA Steps

- [ ] Verify that no errors appear in the JS console
- [ ] Verify all icons look correct during normal regression testing

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).

- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] iOS / native
    - [x] Android / native
    - [x] iOS / Safari
    - [x] Android / Chrome
    - [x] MacOS / Chrome
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product was added in all `src/languages/*` files
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

<details>
<summary><h4>PR Reviewer Checklist</h4>

The reviewer will copy/paste it into a new comment and complete it after the author checklist is completed
</summary>

- [ ] I have verified the author checklist is complete (all boxes are checked off).
- [ ] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [ ] I verified the steps for local testing are in the `Tests` section
    - [ ] I verified the steps for expected offline behavior are in the `Offline steps` section
    - [ ] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [ ] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [ ] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [ ] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [ ] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] iOS / native
    - [ ] Android / native
    - [ ] iOS / Safari
    - [ ] Android / Chrome
    - [ ] MacOS / Chrome
    - [ ] MacOS / Desktop
- [ ] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [ ] I verified that comments were added to code that is not self explanatory
    - [ ] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [ ] I verified any copy / text shown in the product was added in all `src/languages/*` files
    - [ ] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [ ] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn't already exist
    - [ ] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [ ] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

</details>

#### Web

image
image
image
image
image
image
image
image
image
image
image
image
image

#### Mobile Web - Safari

https://user-images.githubusercontent.com/38015950/203447055-c4105028-b7e9-48ba-8d7a-e8da79a9a306.mov

#### Desktop

https://user-images.githubusercontent.com/38015950/203446904-2b2e47ca-8a2c-449f-9b8e-be57f694feee.mov

#### iOS

https://user-images.githubusercontent.com/38015950/203447064-2dc81d11-f329-43c8-ab7d-9d47de1444e0.mov

#### Android

| | | | |
|:--:|:---:|:--:|:---:|
| Screen Shot 2022-11-22 at 5 24 05 PM | Screen Shot 2022-11-22 at 5 21 48 PM | Screen Shot 2022-11-22 at 5 21 43 PM | Screen Shot 2022-11-22 at 5 21 35 PM |

### CODE REVIEW

**Comment by shawnborton** [comment]
I'll work on some of the missing icons and provide them here. There are actually some of the missing ones that are in the .zip file but I think I just did not name them correctly (like flash.svg instead of bolt.svg, gear.svg instead of cog.svg, etc)

**Comment by shawnborton** [comment]
Here's a few more:
[MoreIcons.zip](https://github.com/Expensify/App/files/10026641/MoreIcons.zip)

**Comment by grgia** [comment]
Here's where I'm left:
Quick note- is the hourglass circle being replaced with just the hourglass?

Screen Shot 2022-11-16 at 5 44 10 PM

**Comment by shawnborton** [comment]
apple, android, zoom, shadow, paypal, google-meet, confetti-pop, lets-chat, image-crop-mask don't need to be changed

welcome-screenshot, cascading-cards (both), pin-circle can be deleted (not used anymore)

circle-hourglass can be replaced with just the hourglass icon


So I think that just leaves clipboard, new workspace, example-check (both), and default workspace?

**Comment by shawnborton** [comment]
Here is a new-workspace svg... note that I made it 40x40 as I think that is the correct size for these kinds of things, in case you need to adjust anything code-wise to support that (original looks wider than it is tall, which seems strange): [new-workspace.svg.zip](https://github.com/Expensify/App/files/10036880/new-workspace.svg.zip)


And here is a default workspace avatar: [workspace-default-avatar.svg.zip](https://github.com/Expensify/App/files/10036894/workspace-default-avatar.svg.zip)

Will work on the check images, and TCW is working on the other missing ones (clipboard, link-add).

**Comment by shawnborton** [comment]
Here are new example check images (not the BG is transparent like before, and the height might have changed slightly? I think these should be shown at 200x48?)

[ExampleCheckImages.zip](https://github.com/Expensify/App/files/10036930/ExampleCheckImages.zip)

image

**Comment by grgia** [comment]
Whoops meant to put this here: 
image

**Comment by grgia** [comment]
image
image

Hmm the routing number image is slightly blurry now
Screen Shot 2022-11-18 at 11 33 22 AM

**Comment by shawnborton** [comment]
Here are some new, larger check images to try: [CheckImages.zip](https://github.com/Expensify/App/files/10044359/CheckImages.zip)

**Comment by grgia** [comment]
Could I get those with transparent BG?

**Comment by grgia** [comment]
Screen Shot 2022-11-18 at 11 43 05 AM
Also here's what we're currently left with

**Comment by shawnborton** [comment]
Oops, let's try that again: [CheckImages.zip](https://github.com/Expensify/App/files/10044389/CheckImages.zip)

Will keep you posted on link-copy and clipboard, otherwise I think the rest of your list doesn't need to be updated.

**Comment by grgia** [comment]
Update, looks great!
image

**Comment by shawnborton** [comment]
Agree! 

One thing I am noticing, what about the announce room icon and the admins room icon? Where are those found?

**Comment by grgia** [comment]
Screen Shot 2022-11-18 at 12 10 13 PM

Nice catch, those are in `images/avatars` here!

**Comment by grgia** [comment]
Screen Shot 2022-11-18 at 12 12 24 PM
There's also product illustrations, I know Daniel added a few of these in his WIP PR, but is there an issue to replace all of these with our new illustrations?

**Comment by shawnborton** [comment]
Here's 4 more, though looks like we might only be using two of these in the app... [Uploading MoreIcons.zip…]()

**Comment by grgia** [comment]
Woot! I think that the upload link might be broken, could you try reuploading it?

**Comment by shawnborton** [comment]
Ooops, fixed!

**Comment by shawnborton** [comment]
The last thing is that I need to get you icons for these:
image

**Comment by grgia** [comment]
image
image

Do you want to keep the old navy colors on light mode for the merge? Or is it okay to leave as is?

**Comment by shawnborton** [comment]
I think it's cool to leave them as you have them for now. When we do go to support the new light mode, we can add in the darker avatars.

**Comment by shawnborton** [comment]
Wow, that does feel much bigger. Are they both shown at 20x20?

**Comment by grgia** [comment]
yes both are 20x20. Here's what it looks like at 16 
image

**Comment by shawnborton** [comment]
Could we keep it at 20x20, but try it with this new one that has some negative space around it? [checkmark.svg.zip](https://github.com/Expensify/App/files/10070478/checkmark.svg.zip)

**Comment by grgia** [comment]
image
20x20 with replacement

**Comment by shawnborton** [comment]
Nice, looks good to me!

**Review by Luke9389 — APPROVED** [review]

**Comment by grgia** [comment]
oops wasn't thinking- PR review checklist is done, but let me know if either of you want to do a final pass! Otherwise good to merge

**Comment by Julesssss** [comment]
> We don't need to attach Screenshots/Videos in Reviewer Checklist for this PR?

@aimane-chnaif yes we should

**Comment by grgia** [comment]
@aimane-chnaif did you want to do screenshots for this PR? I'd like to get it merged

**Review by aimane-chnaif — APPROVED** [review]

**Review by nkuoch — APPROVED** [review]

**Review by Julesssss — APPROVED** [review]

**Comment by mvtglobally** [comment]
@grgia @shawnborton @Luke9389 Anything specific you want us to QA in this PR?

File: docs/404.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 permalink: /404.html
 ---
 <div class="centered-content with-lhn">
-    <img class="icon" src="/assets/images/circle-hourglass.svg" />
+    <img class="icon" src="/assets/images/hourglass.svg" />
```
Comment (robertKozik) [inline docs/404.html:5]: `/assets/images/hourglass.svg` was not existing on docs

**Pull request merged**